### PR TITLE
Add a webserver.

### LINKED ISSUE #1: Add an HTTP endpoint able to tell the current ID of the tag on the reader.

### PR DESCRIPTION
**Pull request by Natim**
Fixes #1

### CODE REVIEW

**Comment by Natim** [comment]
ping @sebastienbarbier

File: serial/raspberry_reader_web_server.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                self.send_header('Content-Type', 'application/json')
+                self.end_headers()
+                self.wfile.write(json.dumps({"uid": uid}).encode('utf-8'))
+                return
+
+        self.send_response(404)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+        self.wfile.write(json.dumps({"error": "No tags"}).encode('utf-8'))
+
```
Comment (jlebunetel) [inline serial/raspberry_reader_web_server.py]: Maybe can we keep the "uid" key in the json, even if no tag is detected.

**Review by jlebunetel — CHANGES_REQUESTED** [review]
Maybe can we keep the "uid" key in the json, even if no tag is detected.

**Comment by sebastienbarbier** [comment]
Yeap code works fine, but I am not sure why I would be paranoid about performance and only read a stored value on a GET request. 
Meaning you need to update it when a tag is detected or removed (which would be nice and modular for a pub/sub stream 😁).

**Comment by Natim** [comment]
Yes and then you need to handle concurrency. I guess for this I was kinda lazy :dagger:

**Comment by sebastienbarbier** [comment]
Something is better than nothing 👊 ! And it works, so it ain't stupid 🎉.
You can create an issue to suggest improvement, and merge like this for now probably enough to test and hack around (specially since API will not change after refactoring).

**Comment by Natim** [comment]
> I would be paranoid about performance

Actually after thinking this through I guess reading the tag on request is cheap enough even with loads of request and if we were to add cache we would probably add it at the reverse proxy level.

**Review by jlebunetel — APPROVED** [review]
Perfect!

**Pull request merged**